src/main/java/digital/slovensko/autogram/core/eforms/XDCBuilder.java

```diff
@@ -107,6 +124,19 @@ private static Element createUsedSchemasReferenced(Document document, String xsd
         return element;
     }
 
+    private static Element createUsedSchemasEmbedded(Document document, String xsdSchema, String xsltSchema,
+            XsltParams xsltParams) throws ParserConfigurationException, IOException, SAXException {
+        var element = document.createElement("xdc:UsedSchemasEmbedded");
```

Comment:
neviem ci je uplne dobre takto natvrdo tu davat ten namespace. createElementNS by malo byt safe.

Reply:
Takto sa vytvára aj celý datacontainer aj usedSchemasReferenced, tak som to spravil rovnako. Asi by to bolo viac safe, ale nie je veľmi voči čomu, keďže celé sa to tu vytvára.